app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_path, destination_path = command.split(" ")
+
+    if len(command.split()) == 3 and "mv" in cmd:
```

Comment:
when cmd will "hello_mv" be it will return true. Check difference between in and ==